UI/src/view/linear_asset/manoeuvreStyle.js

```diff
@@ -48,7 +48,7 @@
             new StyleRule().where('zoomLevel', roadLayer.uiState).is(15).use({ pointRadius: 16 })
         ];
 
-        var defaultStyleProvider = new StyleRuleProvider({stroke: {opacity: 0.65, rotation : '${rotation}'}});
+        var defaultStyleProvider = new StyleRuleProvider({stroke: {opacity: 0.65, rotation : ''}});
```

Comment:
You can take out the rotation completely, the stroke object when entering the StyleRuleProvider will not use rotation, only the icon object

```javascript
var defaultStyleProvider = new StyleRuleProvider({stroke: {opacity: 0.65}});
```

Reply:
Applied